edb/lib/std/30-datetimefuncs.edgeql

```diff
@@ -468,18 +467,15 @@ std::`-` (v: std::duration) -> std::duration {
 
 ## String casts
 
-# Casts from text to any sort of date/time types are all STABLE in
-# PostgreSQL, but it may be the case that our restricted versions that
-# prohibit usage of timezone for non-timezone-aware types are actually
-# IMMUTABLE (as the corresponding casts from those types to text).
 CREATE CAST FROM std::str TO std::datetime {
+    # Stable because the input string can be in any time-zone.
```

Comment:
I would also clarify what this means: a dependence on a mutable external timezone database (e.g. tzdata).